app/models/user.js

```diff
@@ -0,0 +1,11 @@
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define('users', {
+    id: { type: DataTypes.INTEGER, allowNull: false, autoIncrement: true, primaryKey: true, unique: true },
+    firstName: { type: DataTypes.STRING, allowNull: false },
+    lastName: { type: DataTypes.STRING, allowNull: false },
+    email: { type: DataTypes.STRING, allowNull: false, unique: true },
+    password: { type: DataTypes.STRING, allowNull: false }
+  });
```

Comment:
When you add the metadata, remember to add the option `underscored: true` here, it lets you avoid having to map them. You add options in an object in the third argument of the `define` function